README.md

```diff
@@ -1,71 +1,108 @@
-# Puppet-lint
+# Puppet Lint
 
 [![Build
 Status](https://secure.travis-ci.org/rodjek/puppet-lint.png)](http://travis-ci.org/rodjek/puppet-lint)
 [![Inline docs](http://inch-ci.org/github/rodjek/puppet-lint.png?branch=master)](http://inch-ci.org/github/rodjek/puppet-lint)
 
-Puppet Lint will test modules and manifests against the recommended Puppet
-style guidelines from the [Puppet Labs style
-guide](http://docs.puppetlabs.com/guides/style_guide.html) as closely as practical. It is not meant to validate syntax. Please use `puppet parser validate` for that.
+Puppet Lint tests modules and manifests against the recommended Puppet style guidelines. It can also help automatically fix style issues in your code.
+
+Puppet Lint tests Puppet code style against the [Puppet language style
+guide](http://docs.puppet.com/puppet/latest/style_guide.html) as closely as practical. It is not meant to validate syntax. Please use Puppet's `puppet parser validate` command for validation.
+
+## Compatibility warning
+
+Release 2.1.0 of Puppet Lint is the last planned version with support for Puppet 3 and Ruby 1.8.7. Future versions will drop support for these versions.
```

Comment:
How about v2 supports Puppet 3 and ruby 1.8.7 and v3 will drop support for that.

That way if someone wants to submit a patch against puppet3, you can always do another v2 release.